src/utils.py

```diff
@@ -68,11 +66,30 @@ def save_issues_to_json_file(issues: List[Issue], object_type: str, output_direc
     output_file_path = f"{output_directory}/{output_file_name}"
 
     # Save a file with correct output
-    with open(output_file_path, 'w', encoding='utf-8') as json_file:
+    with open(output_file_path, 'w', encoding='utf-'
+                                              '8') as json_file:
         json.dump(
-            [issue.to_dict() for issue in issues],
+            state_to_save,
             json_file,
             ensure_ascii=False,
             indent=4)
 
     return output_file_name
+
+
+def load_repository_issue_from_data(directory: str, repository_name: str) -> List[dict]:
```

Comment:
Stop using from_**data**.
Name the source, please.

Reply:
Up to debate

Reply:
https://github.com/AbsaOSS/living-doc-generator/pull/16#discussion_r1680645735